bin/steps/pip-install

```diff
@@ -25,7 +25,7 @@ if [ ! "$SKIP_PIP_INSTALL" ]; then
     fi
 
 
-    /app/.heroku/python/bin/pip install -r "$BUILD_DIR/requirements.txt" --exists-action=w --src=/app/.heroku/src --disable-pip-version-check --no-cache-dir 2>&1 | tee "$WARNINGS_LOG" | cleanup | indent
+    /app/.heroku/python/bin/pip install -r "$BUILD_DIR/requirements.txt" --exists-action=w --src=/app/.heroku/src --disable-pip-version-check --process-dependency-links --no-cache-dir 2>&1 | tee "$WARNINGS_LOG" | cleanup | indent
```

Comment:
ah, earlier version of pip didn't need the `--process-dependency-links` flag to install private dependencies.